### PR TITLE
draft

### PR DESCRIPTION
**Pull request by sczembor**
+ registry implementation 
+ readme

### CODE REVIEW

File: registry/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+use near_sdk::{env, near_bindgen, require, AccountId, PanicOnDefault};
+
+pub mod storage;
+
+#[near_bindgen]
+#[derive(BorshDeserialize, BorshSerialize, PanicOnDefault)]
+pub struct Contract {
+    /// DAO
+    pub authority: AccountId,
+    pub whitelist: UnorderedSet<AccountId>,
```
Comment (robert-zaremba) [inline registry/src/lib.rs]: let's use `LookupSet`

File: registry/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    pub fn blacklist(
+        &mut self,
+        accounts: Vec<AccountId>,
+    ) {
+        let storage_start = env::storage_usage();
+        self.assert_authority();
+        for a in &accounts {
+            if self.whitelist.contains(a) {
+                self.whitelist.remove(a);
+            }
```
Comment (robert-zaremba) [inline registry/src/lib.rs]:
```suggestion
                self.whitelist.remove(a);
```

**Review by robert-zaremba — COMMENTED** [review]
let's do one list with flags.

File: registry/src/lib.rs (hunk 1 above)
Comment (sczembor) [reply registry/src/lib.rs]: DONE

File: registry/src/lib.rs (hunk 2 above)
Comment (sczembor) [reply registry/src/lib.rs]: DONE

**Review by robert-zaremba — APPROVED** [review]

**Pull request merged**